### PR TITLE
[MVP] Rackspace Migration Manager: Move index.md from FAWS folder to rackspace-migration-manager

### LINKED ISSUE #1834: [MVP] Rackspace Migration Manager: Create index.md for landing page
Something similar to https://support.rackspace.com/how-to/dedicated-hosting/

### PR DESCRIPTION
**Pull request by cyrichardson**
folder and update for new product name.

Fixes #1834 

Replaces PR #1835.

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-940ec82ea7/how-to/)

**Comment by cyrichardson** [comment]
Fixes #1834

**Comment by cyrichardson** [comment]
Release is currently planned for mid to late July.

**Comment by cyrichardson** [comment]
Adding blocked status until product launch.

**Comment by cyrichardson** [comment]
Moving to Blocked status until product launch.

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-753d8285c5/how-to/)

**Comment by cyrichardson** [comment]
@trentgillaspie Please review when you have time. Thanks!

File: content/rackspace-migration-manager/index.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Many businesses are seeking assistance from certified experts to get the most
+out of Amazon Web Services (AWS). Migration, architecture design, security,
+and operations can take your focus away from your core business.
+
+The Rackspace Migration Manager is the answer for businesses migrating from Rackspace to AWS. The Rackspace Migration Manager blends technology, automation, and expertise to move from the Rackspace Public Cloud to AWS and 24x7x365 operations backed by AWS certified engineers and architects.
+
+Rackspace provides the tooling, process, and support that enables you to
+migrate from Rackspace Public Cloud to AWS while maintaining the Fanatical
+Support&reg; you expect from Rackspace. Rackspace also provides Rackspace
```
Comment (trentgillaspie) [inline content/rackspace-migration-manager/index.md]: "Rackspace can ALSO provide Rackspace"

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-cc3f43fd07/how-to/)

**Comment by cyrichardson** [comment]
Fixes #1834

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-94155b27e8/how-to/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-2d7591a8d5/how-to/)

**Comment by cyrichardson** [comment]
Ready for MVP launch

**Comment by cyrichardson** [comment]
Closing in favor of https://jira.rax.io/browse/INFODEV-424

**Pull request closed without merging**